apps/prairielearn/src/ee/auth/lti13/lti13Auth.html.ts

```diff
@@ -46,3 +46,52 @@ export const Lti13Test = ({
     </html>
   `.toString();
 };
+
+export const Lti13Iframe = ({
+  resLocals,
+  target_url,
+}: {
+  resLocals: Record<string, any>;
+  target_url: string;
+}) => {
+  return html`
+    <!doctype html>
+    <html lang="en">
+      <head>
+        ${HeadContents({ resLocals, pageTitle: 'LTI 1.3 redirect' })}
+        <script>
+          window.parent.postMessage(
+            {
+              subject: 'lti.frameResize',
```

Comment:
This isn't necessary but it makes the Canvas page cleaner since it gives a big viewport for the iFrame and we use something very narrow.

Reply:
Removing this because it's unnecessary (and probably Canvas specific, which I'd like to minimize at the moment until we have ways to differentiate between platforms).